gateway/src/apicast/policy/logging/logging.lua

```diff
@@ -0,0 +1,37 @@
+--- Logging policy
+
+local _M  = require('apicast.policy').new('Logging Policy')
+
+local new = _M.new
+
+local default_enable_access_logs = true
+
+-- Returns the value for ngx.var.enable_acess_logs
+local function enable_access_log_value(enabled)
+  return (enabled and '1') or '0'
```

Comment:
Better to use dispatch table for this. Conditionals are not JITable.

Reply:
In this case, it doesn't matter much, because it's only called from `new()`.
But I changed it because as a general rule we should do this, and I tend to forget about this :sweat_smile: